provider/examples/resources/sdc/main.tf

```diff
@@ -0,0 +1,19 @@
+module "vpc" {
+  source = "github.com/CiscoDevNet/cdo-automation//modules/aws_vpc"
+}
+
+
+module "sdc_in_aws" {
```

Comment:
In an example on using sdc, we really should be using `resource "cdo_sdc"` -- otherwise it's confusing.